include/RED4ext/Scripting/Natives/redTagList.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <RED4ext/CName.hpp>
+#include <RED4ext/Common.hpp>
+#include <RED4ext/DynArray.hpp>
+
+namespace RED4ext
+{
+namespace red
+{
+struct TagList
+{
+    static constexpr const char* NAME = "redTagList";
+    static constexpr const char* ALIAS = NAME;
+
+    operator bool() const noexcept;
+
+    [[nodiscard]] bool IsEmpty() const;
+    [[nodiscard]] bool HasTag(CName aTag) const;
+    [[nodiscard]] bool HasTags(const DynArray<CName>& aTags) const;
+    [[nodiscard]] bool HasTags(const TagList& aTags) const;
```

Comment:
What about
```suggestion
    [[nodiscard]] bool Contains(CName aTag) const;
    [[nodiscard]] bool Contains(const DynArray<CName>& aTags) const;
    [[nodiscard]] bool Contains(const TagList& aTags) const;
```
?
Same for `Add` and `Remove`.